### ISSUE TITLE
AttributeError: 'list' object has no attribute 'fit'

### ISSUE THREAD

**Issue by DmitriR** [paulgb/sklearn-pandas#39]
Code straight from the docs page. Examples until that point work fine.

```
print(sys.version)
print('sklearn_pandas', sklearn_pandas.__version__)

mapper3 = DataFrameMapper([
     (['age'], [sklearn.preprocessing.Imputer(),
                sklearn.preprocessing.StandardScaler()])])
data_3 = pd.DataFrame({'age': [1, np.nan, 3]})
mapper3.fit_transform(data_3)
```
gives me 

```
3.4.3 |Anaconda 2.3.0 (x86_64)| (default, Mar  6 2015, 12:07:41) 
[GCC 4.2.1 (Apple Inc. build 5577)]
sklearn_pandas 0.0.10

---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-97-068003ccd118> in <module>()
      6                 sklearn.preprocessing.StandardScaler()])])
      7 data_3 = pd.DataFrame({'age': [1, 2, 3]})
----> 8 mapper3.fit_transform(data_3)

/anaconda/anaconda3/anaconda/lib/python3.4/site-packages/sklearn/base.py in fit_transform(self, X, y, **fit_params)
    431         if y is None:
    432             # fit method of arity 1 (unsupervised transformation)
--> 433             return self.fit(X, **fit_params).transform(X)
    434         else:
    435             # fit method of arity 2 (supervised transformation)

/anaconda/anaconda3/anaconda/lib/python3.4/site-packages/sklearn_pandas/__init__.py in fit(self, X, y)
    111         for columns, transformer in self.features:
    112             if transformer is not None:
--> 113                 transformer.fit(self._get_col_subset(X, columns))
    114         return self
    115 

AttributeError: 'list' object has no attribute 'fit'
```

**Comment by dukebody** [comment]
Duplicate of https://github.com/paulgb/sklearn-pandas/issues/38

**Comment by DmitriR** [comment]
Fix as explained in #38 : 
`pip install --upgrade git+https://github.com/paulgb/sklearn-pandas.git@master`

Thanks @dukebody

**Comment by dukebody** [comment]
We've released a new version in PyPI, 0.0.12, that allow list transformers.
